src/Common/src/System/Diagnostics/Debug.Unix.cs

```diff
@@ -8,108 +8,118 @@ namespace System.Diagnostics
 {
     // Intentionally excluding visibility so it defaults to internal except for
     // the one public version in System.Diagnostics.Debug which defines
-    // another version of this partial class with the public visibility 
+    // another version of this partial class with the public visibility
     static partial class Debug
     {
-        private static string NewLine => "\n"; 
+#if !MONO
```

Comment:
Yes but then with the mono configuration, since we compile both `Debug.Unix.cs` and `Debug.Windows.cs` (since `Debug.Mono.cs` is just a glue betweem unix and windows code), we will have a duplicated symbol Debug.NewLine with `Debug.Windows.cs/Debug.NewLine` and `Debug.Unix.cs/Debug.NewLine`.